subnet/subnet.go

```diff
@@ -32,8 +32,10 @@ type LeaseAttrs struct {
 
 type Lease struct {
 	Subnet     ip.IP4Net
-	Attrs      *LeaseAttrs
+	Attrs      LeaseAttrs
 	Expiration time.Time
+
+	asof uint64
```

Comment:
I do not follow how `asof` is being used. I initially thought it was passed to `updateSubnet` and then used in `etcd.SetOptions` to confirm the previous modified index of the key. But all calls to `updateSubnet` in `local_manager` pass `asof=0`.